samples/dml.js

```diff
@@ -630,6 +630,74 @@ async function updateUsingBatchDml(instanceId, databaseId, projectId) {
   // [END spanner_dml_batch_update]
 }
 
+async function executeSqlWithCustomTimeoutAndRetrySettings(
+  instanceId,
+  databaseId,
+  projectId
+) {
+  // [START spanner_set_custom_timeout_and_retry]
+  // Imports the Google Cloud client library
+  const {Spanner} = require('@google-cloud/spanner');
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const projectId = 'my-project-id';
+  // const instanceId = 'my-instance';
+  // const databaseId = 'my-database';
+
+  // Creates a client
+  const spanner = new Spanner({
+    projectId: projectId,
+  });
+  const DEADLINE_EXCEEDED_STATUS_CODE = 4;
```

Comment:
Why we don't use `grpc.status.DEADLINE_EXCEEDED`? Instead, we use hardcoded values here.

Reply:
You can do:
```
const {grpc} = require('google-gax');
...
retryCodes: [
  grpc.status.DEADLINE_EXCEEDED,
  grpc.status.UNAVAILABLE,
],
```

Reply:
It gives below error as we don't have direct dependency of `google-gax` in sample
`##[error]  17:24  error  "google-gax" is extraneous  node/no-extraneous-require`

Reply:
Oh yeah I remember that happening for a separate issue. I don't know of a workaround. It's ok to leave this as is for now. Thanks for trying.